src/components/AppBar/index.tsx

```diff
@@ -1,27 +1,21 @@
 import React from "react";
 import { Button, Popover } from "antd";
-import { useWallet } from "../../contexts/wallet";
-import { CurrentUserBadge } from "../CurrentUserBadge";
 import { SettingOutlined } from "@ant-design/icons";
 import { Settings } from "../Settings";
 import { LABELS } from "../../constants";
-import { ConnectButton } from "../ConnectButton";
+import {
+  WalletDisconnectButton,
+  WalletMultiButton,
+} from "@solana/wallet-adapter-ant-design";
+import { useWallet } from "@solana/wallet-adapter-react";
 
 export const AppBar = (props: { left?: JSX.Element; right?: JSX.Element }) => {
   const { connected } = useWallet();
-
   const TopBar = (
     <div className="App-Bar-right">
-      {connected ? (
-        <CurrentUserBadge />
-      ) : (
-        <ConnectButton
-          type="text"
-          size="large"
-          allowWalletChange={true}
-          style={{ color: "#2abdd2" }}
-        />
-      )}
+      <WalletMultiButton />
+      <div style={{ margin: 5 }} />
+      {connected ? <WalletDisconnectButton color="secondary" /> : null}
```

Comment:
Probably the styles of these buttons will need to be changed to match this project. A PR to make them customizable in the wallet-adapter/ant-design package is welcome.